components/Auth/Login.tsx

```diff
@@ -0,0 +1,173 @@
+
+      <VStack w="100%" maxW="xs" as="form">
+        <Controller
+          control={methods.control}
+          name="email"
+          render={({ field, fieldState }) => {
+            return (
+              <VStack w="100%">
+                <Input
+                  borderColor={fieldState.error ? 'red' : 'inherit'}
+                  type="email"
+                  value={field.value}
+                  placeholder="Email Address"
+                  onChange={field.onChange}
+                  onBlur={field.onBlur}
+                  onKeyDown={(e) => {
+                    if (e.key == 'Enter') {
+                      methods.setFocus('password')
+                    }
+                  }}
+                />
+                {fieldState.error && (
+                  <Text w="100%" color="red">
+                    {fieldState.error.message}
+                  </Text>
+                )}
+              </VStack>
+            )
+          }}
+        />
+        <Controller
+          control={methods.control}
+          name="password"
+          render={({ field, fieldState }) => {
+            return (
+              <VStack w="100%">
+                <InputGroup>
+                  <Input
+                    ref={field.ref}
+                    onBlur={field.onBlur}
+                    borderColor={fieldState.error ? 'red' : 'inherit'}
+                    type={showPassword ? 'text' : 'password'}
+                    value={field.value}
+                    placeholder="Password"
+                    onChange={field.onChange}
+                    onKeyDown={(e) => {
+                      if (e.key == 'Enter') {
+                        methods.handleSubmit(onSubmit)()
+                      }
+                    }}
+                  />
+                  <InputRightElement
+                    // eslint-disable-next-line react/no-children-prop
+                    children={
+                      <Button
+                        variant="ghost"
+                        backgroundColor="inherit"
+                        px={6}
+                        mr={2}
+                        fontWeight="light"
+                        onClick={() => setShowPassword(!showPassword)}
+                      >
+                        {showPassword ? 'Hide' : 'Show'}
+                      </Button>
+                    }
+                  />
+                </InputGroup>
+                {fieldState.error && (
+                  <Text w="100%" color="red">
+                    {fieldState.error.message}
+                  </Text>
+                )}
+              </VStack>
+            )
+          }}
+        />
+
+        <Button w="100%" onClick={methods.handleSubmit(onSubmit)}>
+          Sign In
+        </Button>
+      </VStack>
+
+      <VStack>
+        <Text
+          fontWeight="medium"
+          cursor={'pointer'}
+          _hover={{ textDecor: 'underline' }}
+        >
+          Forgot your Password?
+        </Text>
```

Comment:
I actually added a [`clickable` `Text` variant](https://github.com/MutualAidNYC/services-lists/blob/main/components/theme.ts#L46) for this 🤓